.github/CONTRIBUTING.md

```diff
@@ -1 +1,4 @@
 # Guide for Contributions
+
+
+To contibueute please visit this [link](https://nowhere.xyz)
```

Comment:
Bad grammar yo!

Reply:
```suggestion
To contribute please visit this [link](https://nowhere.xyz)
```

Reply:
@brooksseahorn hey, is the above better?  if it is, go ahead and commit those changes

Reply:
Looking good!